src/coreclr/vm/threads.h

```diff
@@ -2716,7 +2716,7 @@ class Thread
                                                    // stack crawl. Eventually need to always do this but it
                                                    // breaks the debugger right now.
 
-    #define LIGHTUNWIND                     0x0020 // allow using cache schema (see StackwalkCache class)
```

Comment:
There is confusion about what's called LIGHTUNWIND vs. QUICKUNWIND. The implementation of the LIGHTUNWIND was under HAS_QUICKUNWIND ifdef and otherwise used Quick Unwind names.

The distinction between LIGHTUNWIND vs. QUICKUNWIND seems to important for Windows x86 unwinding in abnormal situations, so I am going to keep it, but I am going to rename everything related to light unwind to be called light unwind.